src/commands/fun/dicksize.sakuria.ts

```diff
@@ -1,51 +1,130 @@
 import { randomDickSize } from "../../logic/logic.sakuria";
 import { defineCommand } from "../../types";
 import { Readable } from "stream";
+import { User } from "discord.js";
+import SakuriaEmbed, { createErrorEmbed, createInlineBlankField } from "src/sakuria/SakuriaEmbed.sakuria";
 
 export default defineCommand({
   name: "dicksize",
   description: "Tell's you your dicksize or battle against someone else's dicksize!",
   requiresProcessing: false,
   execute: async (message) => {
-    if (message.mentions.members && message.mentions.members.size !== 0) {
-      const dicksize = randomDickSize();
-      const enemyDicksize = randomDickSize();
+    const mentionedUsers = message.mentions.users;
 
-      if (dicksize + enemyDicksize > 1900) {
+    if (mentionedUsers.size) {
+      if (mentionedUsers.size > 20) {
         return {
-          content:
-            `This battle of the dongs is too much to just say as is, so here's the brief:\n` +
-            `${message.author}: ${dicksize}cm\n` +
-            `${message.mentions.members.first()}: ${enemyDicksize}cm` +
-            `diff: ${Math.abs(dicksize - enemyDicksize)}cm` +
-            `winner: ${dicksize > enemyDicksize ? message.author : message.mentions.members.first()}`,
+          embeds: [createErrorEmbed("For you guys' healthiness, only 20 people can join the fight at once(including you).")]
+        };
+      }
+
+      const authorDickSize = randomDickSize();
+      const allDicks: [User, number][] = [[message.author, authorDickSize]];

```

Comment:
you should turn this into an object instead of a tuple
and use the user ID as they not the entire user instance, performance++;